tests/features/content_types/article.feature

```diff
@@ -20,3 +20,20 @@ Feature: Article content type
     And I should see the heading "EC decides tax advantages for Fiat are illegal"
     And I should see the text "Commissioner states tax rulings are not in line with state aid rules."
     And I should see the text "Published by Anonymous"
+
+  @api @europa_theme
```

Comment:
May be, we can create 3 files, one for all theme, one for ec_resp and another for europa - it's not the case for this file.